src/app/SignupViewFactory.java

```diff
@@ -20,6 +20,14 @@ public class SignupViewFactory {
     private SignupViewFactory() {
     }
 
+    /**
+     * Creates a SignupView
+     * @param viewManagerModel
+     * @param loginViewModel
+     * @param signupViewModel
+     * @param userDataAccessObject
+     * @return
```

Comment:
Small nit pick, but in the @return line, add what it returns which is obviously a SingupView.

Reply:
Changed